esmlab/__init__.py

```diff
@@ -1,11 +1,18 @@
 #!/usr/bin/env python
+import sys
+from .config import set_options, get_options
```

Comment:
Can you move these two lines underneath the module doc-string line?

Reply:
Done.